docker-compose.yml

```diff
@@ -20,6 +20,7 @@ services:
             - ./otter:/app/otter
             - ./scripts:/app/scripts
             - ./twisted:/app/twisted
+            - ./schema:/app/schema
```

Comment:
This reminds me that there should really be a second `docker-compose.yml` that actually ships all the code in the image rather than depending on volumes.  This is better / quicker for development, but not a best practice for deployment.